mcs/class/corlib/corefx/GlobalizationSupport.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Globalization
+{
+	static class GlobalizationSupport
```

Comment:
Please remove this unreachable file

Reply:
It is only unreachable on WASM.

Reply:
sure but for the rest it's true so no-op